internal/rbd/controllerserver.go

```diff
@@ -1559,3 +1559,35 @@ func (cs *ControllerServer) ControllerExpandVolume(
 		NodeExpansionRequired: nodeExpansion,
 	}, nil
 }
+
+func (cs *ControllerServer) ControllerPublishVolume(
+	ctx context.Context,
+	req *csi.ControllerPublishVolumeRequest,
+) (*csi.ControllerPublishVolumeResponse, error) {
+	// A dummy publish implementation to mimic a successful attach operation being a NOOP
+	if req.VolumeId == "" {
```

Comment:
use functions like `req.GetVolumeId()` instead of accessing the attribute directly.